BraveWallet/Crypto/NetworkSelectionView.swift

```diff
@@ -0,0 +1,336 @@
+          }
+        }
+      }
+      Section(content: {
+        ForEach(store.secondaryNetworks) { presentation in
+          NetworkRowView(
+            presentation: presentation,
+            selectedNetwork: networkStore.selectedChain
+          )
+          .onTapGesture {
+            selectNetwork(presentation.network)
+          }
+        }
+      }, header: {
+        WalletListHeaderView(title: Text(Strings.Wallet.networkSelectionSecondaryNetworks))
+      })
+    }
+    .listStyle(.insetGrouped)
+    .navigationTitle(Strings.Wallet.networkSelectionTitle)
+    .navigationBarTitleDisplayMode(.inline)
+    .toolbar {
+      ToolbarItemGroup(placement: .cancellationAction) {
+        Button(action: { presentationMode.dismiss() }) {
+          Text(Strings.cancelButtonTitle)
+            .foregroundColor(Color(.braveOrange))
+        }
+      }
+    }
+    .onAppear {
+      store.update()
+    }
+    .background(
+      NavigationLink(
+        isActive: Binding(
+          get: { store.detailNetwork != nil },
+          set: { if !$0 { store.detailNetwork = nil } }
+        ),
+        destination: {
+          if let detailNetwork = store.detailNetwork {
+            NetworkSelectionDetailView(
+              networks: detailNetwork.subNetworks,
+              selectedNetwork: networkStore.selectedChain,
+              selectedNetworkHandler: { network in
+                selectNetwork(network)
+              }
+            )
+          }
+        },
+        label: { EmptyView() }
+      )
+    )
+    .background(
+      Color.clear
+        .alert(
+          isPresented: $store.isPresentingNextNetworkAlert
+        ) {
+          Alert(
+            title: Text(String.localizedStringWithFormat(Strings.Wallet.createAccountAlertTitle, store.nextNetwork?.shortChainName ?? "")),
+            message: Text(Strings.Wallet.createAccountAlertMessage),
+            primaryButton: .default(Text(Strings.yes), action: {
+              store.handleCreateAccountAlertResponse(shouldCreateAccount: true)
+            }),
+            secondaryButton: .cancel(Text(Strings.no), action: {
+              store.handleCreateAccountAlertResponse(shouldCreateAccount: false)
+            })
+          )
+        }
+    )
+    .background(
+      Color.clear
+        .sheet(
+          isPresented: $store.isPresentingAddAccount
+        ) {
+          NavigationView {
+            AddAccountView(keyringStore: keyringStore, preSelectedCoin: store.nextNetwork?.coin)
+          }
+          .onDisappear {
+            Task { @MainActor in
+              await store.handleDismissAddAccount()
+            }
+          }
+        }
+    )
+  }
+  
+  private func selectNetwork(_ network: BraveWallet.NetworkInfo) {
+    Task { @MainActor in
+      if await store.selectNetwork(network: network) {
+        presentationMode.dismiss()
+      }
+    }
+  }
+}
+
+/*
+#if DEBUG
+struct NetworkSelectionView_Previews: PreviewProvider {
```

Comment:
Can be deleted?

Reply:
Removed in [c29f314](https://github.com/brave/brave-ios/pull/5706/commits/c29f314f45ae50559626c245a8c40fe2dbdfd34e)